### PR TITLE
Apply the cookiecutter to Supervisor config files

### PR DESCRIPTION
**Pull request by seanh**
Depends on https://github.com/hypothesis/cookiecutters/pull/91

### CODE REVIEW

File: conf/supervisord.conf [hunk 1]
```diff
@@ -6,24 +6,24 @@ logfile_maxbytes=0
...
 stderr_events_enabled=true
+stdout_logfile=NONE
+stderr_logfile=NONE
 
 [program:checkmate-beat]
 command=celery -b %(ENV_CHECKMATE_BROKER_URL)s -A h_periodic.checkmate_beat beat --pidfile=checkmate-celerybeat.pid --loglevel INFO
 stdout_events_enabled=true
 stderr_events_enabled=true
-stopsignal = KILL
-stopasgroup = true
```
Comment (seanh) [inline conf/supervisord.conf:19]: The cookiecutter has actually changed these two settings in h-periodic for the production `checkmate-beat` and `lms-beat` programs. This makes the `checkmate-beat` and `lms-beat` programs the same as the `h-beat` one. I believe this is correct: the `stopsignal` and `stopasgroup` settings are meant for dev not prod. And programs in prod are apparently supposed to have the `stdout_logfile=NONE` and `stderr_logfile=NONE` settings. These settings just turn off Supervisor's default behaviour of writing the programs logs to files (see [Supervisor's docs](http://supervisord.org/configuration.html)) which makes sense: we take the logs from stdout and stderr directly into Papertrail, we wouldn't read any log files created on the VM's filesystem.

File: conf/supervisord.conf [hunk 2]
```diff
@@ -6,24 +6,24 @@ logfile_maxbytes=0
...
+stdout_logfile=NONE
+stderr_logfile=NONE
 
 [program:lms-beat]
 command=celery -b %(ENV_LMS_BROKER_URL)s -A h_periodic.lms_beat beat --pidfile=lms-celerybeat.pid --loglevel INFO
 stdout_events_enabled=true
 stderr_events_enabled=true
-stopsignal = KILL
-stopasgroup = true
```
Comment (seanh) [inline conf/supervisord.conf:26]: I believe this change is correct (see comment above)

File: conf/supervisord.conf (hunk 1 above)
Comment (jon-betts) [reply conf/supervisord.conf:19]: Yeah, probably copy based based nonsense?

**Review by jon-betts — APPROVED** [review]
Looks good. It also looks like the templating has caught some mistakes here as well.

**Pull request merged**